### PR TITLE
39 of 39 green.

### PR DESCRIPTION
**Pull request by TStrothjohann**

### CODE REVIEW

File: lib/questions.rb [hunk 1]
```diff
@@ -187,12 +258,23 @@ def word_count_a_file(file_path)
 # called call_method_from_string('foobar')
 # the method foobar should be invoked
 def call_method_from_string(str_method)
+  str_method()
```
Comment (TStrothjohann) [inline lib/questions.rb]: I guess this error is part of this kata's solution...

Comment (spike01) [inline lib/questions.rb]: This isn't JavaScript!

Comment (spike01) [inline lib/questions.rb]: `send(str_method.to_sym)`
...but just checked yours and it worked, because it'll raise a NameError. The test needs to be better.

**Comment by spike01** [comment]
:checkered_flag: 

Congrats on completing.

**Pull request closed without merging**